src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/ProbationStatus.java

```diff
@@ -4,7 +4,9 @@
 
 public enum ProbationStatus {
     CURRENT("Current"),
-    PREVIOUSLY_KNOWN("Previously known");
+    PREVIOUSLY_KNOWN("Previously known"),
+    NOT_SENTENCED("Not sentenced"),
```

Comment:
Apologies, I failed to mention on the ticket but we want to map `Not sentenced` to `No record` in the front end. I had to add `NOT_SENTENCED` into the community-api status so that the interface accurately reflects the reality (i.e. we do have a record, it's just they haven't been sentenced yet) but for the purposes of prepare-a-case we don't make a distinction between `No record` and `Not sentenced`

Reply:
Done - will return "no record" when serialising NOT_SENTENCED to Json